ElementX/Sources/Services/Analytics/PostHogAnalyticsClient.swift

```diff
@@ -73,6 +85,20 @@ class PostHogAnalyticsClient: AnalyticsClientProtocol {
                                                                    numSpaces: userProperties.numSpaces ?? pendingUserProperties.numSpaces)
     }
     
+    func updateSuperProperties(_ updatedProperties: AnalyticsEvent.SuperProperties) {
+        guard let currentProperties = superProperties else {
+            superProperties = updatedProperties
+            return
+        }
+        
+        superProperties = AnalyticsEvent.SuperProperties(appPlatform: updatedProperties.appPlatform ??
+            currentProperties.appPlatform,
+            cryptoSDK: updatedProperties.cryptoSDK ??
+                currentProperties.cryptoSDK,
+            cryptoSDKVersion: updatedProperties.cryptoSDKVersion ??
+                currentProperties.cryptoSDKVersion)
```

Comment:
The preview looks dumb with the wrapping, but would be nice to keep both sides of the `??` on the same line like we do for `UserProperties`.
```suggestion
        superProperties = AnalyticsEvent.SuperProperties(appPlatform: updatedProperties.appPlatform ?? currentProperties.appPlatform,
                                                         cryptoSDK: updatedProperties.cryptoSDK ?? currentProperties.cryptoSDK,
                                                         cryptoSDKVersion: updatedProperties.cryptoSDKVersion ?? currentProperties.cryptoSDKVersion)
```

Reply:
can't make swiftformat happy with any other formatting